src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java

```diff
@@ -142,12 +142,16 @@ public void restartFloodlight(String region) {
 
     @Override
     public List<FloodlightResourceAddress> knockoutSwitch(Switch sw, List<String> regions) {
+        throw new RuntimeException("This method is not allowed to run on hardware environment. " +
+                "See https://github.com/telstra/open-kilda/issues/5074 for details");
+        /* Don't delete this code. It's commented out to allow to build the class
```

Comment:
The code will be available in VCS history. It's easy to restore deleted code from previous versions, so no need for commenting out unreachable code? Is there some additional value in this comment?

Reply:
Fixed.